src/DI/ContainerLoader.php

```diff
@@ -108,7 +108,7 @@ protected function generate($class, $generator)
 	{
 		$compiler = new Compiler;
 		$compiler->getContainerBuilder()->setClassName($class);
-		$code = call_user_func_array($generator, [& $compiler]);
+		$code = $generator(...[& $compiler]);
```

Comment:
removing `call_user_func_*` will not work in all cases, e.g. if `$generator` is string.